tests/integration/BUILD.bazel

```diff
@@ -128,7 +122,7 @@ py_gapic_library(
         "python-gapic-name=logging",
         "autogen-snippets",
     ],
-    transport = "grpc+rest",
+    transport = "grpc",
```

Comment:
Please add a comment explaining why REST is removed

For example, 
- Link to https://github.com/googleapis/gapic-generator-python/issues/1728
- Link to https://github.com/googleapis/googleapis/blob/e85662e798a0a9495a035839f66d0c037c481e2c/google/logging/v2/BUILD.bazel#L201 which shows that REST isn't generated for google/logging/v2